CONTRIBUTING.md

```diff
@@ -0,0 +1,7 @@
+# CONTRIBUTING
```

Comment:
Maybe a starting point is to add notes here about how to contribute to the template repo itself?

Reply:
document details (from our standards guide) forthcoming! - 🚀